Sources/TuistKit/Services/SecretService.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import Signals
+import TSCBasic
+import TuistGenerator
+import TuistSupport
+
+final class SecretService {
+    private let secureStringGenerator: SecureStringGenerating
+
+    convenience init() {
+        self.init(secureStringGenerator: SecureStringGenerator())
+    }
+
+    init(secureStringGenerator: SecureStringGenerating) {
```

Comment:
Just out of curiosity - is there a reason why you did not add the default value `SecureStringGenerator()` as that's how it's done throughout most of the codebase?